drag_and_drop_v2/templates/html/drag_and_drop_edit.html

```diff
@@ -116,6 +116,7 @@
                 <h3>{% trans "Items" %}</h3>
             </header>
             <section class="tab-content">
+                <div class="xblock-editor-error-messages" aria-live="polite" tabindex="0"></div>
```

Comment:
Doesn't this div need to be present on all the tabs? Right now it is just within "Items".